crates/holochain_conductor_api/src/config/conductor/passphrase_service_config.rs

```diff
@@ -12,22 +12,33 @@ use std::path::PathBuf;
 #[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
 #[serde(tag = "type", rename_all = "lowercase")]
```

Comment:
lowercaseisnotquiteasreadable

```suggestion
#[serde(tag = "type", rename_all = "snake_case")]
```